src/routes/(authenticated)/my-account/+page.svelte

```diff
@@ -57,7 +58,7 @@
 		if (data.redirectUrl) {
 			goto(data.redirectUrl);
 		}
-		//TODO notify user of success
+		toast.push(m.saved() + ' ✅');
```

Comment:
Because I am just seeing this and I am wondering: Would it be possible to use FA-Icons here instead of emojies?

Reply:
Ah that probably would be better. Not sure how to properly embed this into the toas though. I wanted this to be a quick thing so I can take this out if you want?